expediagroup/sdk/core/client/api.py

```diff
@@ -124,9 +123,15 @@ def call(
                 timeout=self.request_timeout,
             )
 
+        should_mask_logs: bool = LOG.getEffectiveLevel() > logging.DEBUG or LOG.getEffectiveLevel() == logging.NOTSET
```

Comment:
We discussed this again. Let's mask the fields regardless of the log level, please.

Reply:
Done.